### DISCUSSION TITLE
Add filament left to be consumed in print.

### DISCUSSION THREAD

**Discussion by texeryone** [OrcaSlicer/OrcaSlicer#6046] [Ideas]
On the Device page, it displays the following: (Bambu Lab X1C on 2.11
Plate name
Model name, and print settings
Print status

Print time & filament that will be consumed for the whole print

Where it shows how much filament will be used for the whole print plate that the printer is actively printing, is it possible that there could also be something like: (using 420 as an example)
" Total filament to be used: 420g
   Total filament printed: 102g
   Total filament left to print: 318g"
   or something similar like:
   "Total filament: 420g
     Used so far: 102g"

![AUTOMATIC+LUBE+HELPER+V1+WO+LOGO+SMOOTH - OrcaSlicer 7_6_2024 9_34_10 PM](https://github.com/SoftFever/OrcaSlicer/assets/117549957/1eed3cf3-6a76-46e6-9333-0dc85cf864f9)
Circled in Blue is the filament used that I am talking about

**Comment by johnny-bit** [comment]
Since there's time remaining, filament remaining would be a good thing to have... But then with multi-material it'd have to be split into different material (or overall printed and with details in pop-up hover)
